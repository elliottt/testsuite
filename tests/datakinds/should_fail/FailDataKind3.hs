{-# LANGUAGE DataKinds #-}
module FailDataKind3 where

-- Fails because of no -XPolyKinds
data kind Foo a = Bar
