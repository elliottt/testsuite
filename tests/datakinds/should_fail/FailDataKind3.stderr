
FailDataKind3.hs:5:1:
    Illegal polymorphic `data kind` declaration (use -XPolyKinds to enable)
